smoke-tests/designer/features/linkingPages.feature

```diff
@@ -14,6 +14,7 @@ Feature: Linking Pages
     When I link ths page to link from the "/first-page"
     Then my page is created with a link to the page
 
+  @flaky
```

Comment:
If it's flaky, is it best to just remove until it can be looked at? Otherwise any result from it can't be trusted?

Reply:
@jonnywyatt the flakiness is due to the minification/uglification of the JS which changes the react selector, I am requesting a css locator as none exists presently.